config.json

```
@@ -135,6 +135,66 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
-  "tags": []
-}
+  "key_features": [
+    {
+      "icon": "features-oop",
+      "title": "Modern",
+      "content": "VB.NET is a modern, fast-evolving language."
+    },
+    {
+      "icon": "features-strongly-typed",
+      "title": "Cross-platform",
+      "content": "VB.NET runs on almost any platform and chipset."
+    },
+    {
+      "icon": "features-functional",
+      "title": "Multi-paradigm",
+      "content": "VB.NET is primarily an object-oriented language, but also has lots of functional features."
+    },
+    {
+      "icon": "features-lazy",
+      "title": "General purpose",
+      "content": "VB.NET can be used for a wide variety of workloads, like websites, console applications, and even games."
```

This one has 4 characters too many, maximum length is 100

---

```suggestion
      "content": "VB.NET is used for a wide variety of workloads, like websites, console applications, and even games."
```

This edit would put it at exactly 100 characters.

---

Okay, made that change